packages/opentelemetry-plugin-https/package.json

```diff
@@ -6,10 +6,11 @@
   "types": "build/src/index.d.ts",
   "repository": "open-telemetry/opentelemetry-js",
   "scripts": {
-    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.ts'",
+    "test": "nyc ts-mocha -p tsconfig.json test/**/*/*.test.ts",
```

Comment:
is this correct ?, aren't there too many "stars" 
you have
`test/**/*/*.test.ts`
but it should be 
`test/**/*.test.ts`

Reply:
Seems I copied a bug from http... corrected in http and https plugin.